include/verilated_imp.h

```diff
@@ -413,6 +413,11 @@ class VerilatedImp final {
         const VerilatedLockGuard lock(s_s.v.m_hierMapMutex);
         s_s.v.m_hierMap[fromp].push_back(top);
     }
+    static void hierarchyClear() VL_MT_SAFE {
```

Comment:
Not actually sure what to do wrt multi-threading here.  But I assumed it was safe to copy `hierarchyAdd()`.

Reply:
This seems right.